addons/pos_restaurant/static/src/app/floor_screen/floor_screen.js

```diff
@@ -383,6 +383,7 @@ export class FloorScreen extends Component {
                 }
             }
             const order = this.pos.get_order();
+            order._updateRewards();
```

Comment:
can't call this here. This results to crash if pos_loyalty is not installed.

why do the rewards disappear when getting out of the table?